File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.py
import re

def solution(my_str, n):
    # 1.
    # p = re.compile(f'.{{1,{n}}}')
    # return p.findall(my_str)
    
    # 2.
    return [my_str[i:i+n] for i in range(0, len(my_str), n)]

File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.js
function solution(my_str, n) {
    // 1. 정규표현식
    return my_str.match(new RegExp(`.{1,${n}}`, 'g'));
    
    // 2. 정규표현식 안쓰고
    // let result = []
    // for (let i = 0; i < my_str.length; i += n) {
    //     result.push(my_str.slice(i, i + n))
    // }
    // return result
}